test/parallel/test-stream-writable-clear-buffer.js

```diff
@@ -1,33 +1,32 @@
 'use strict';
-const common = require('../common');
+require('../common');
 const Stream = require('stream');
 // This test ensures that the _writeableState.bufferedRequestCount and
-// the actual buffered request count are the same
+// the actual buffered request count are the same.
 const assert = require('assert');
 
 class StreamWritable extends Stream.Writable {
   constructor() {
     super({ objectMode: true });
   }
 
-  // We need a timeout like on the original issue thread
-  // otherwise the code will never reach our test case
-  // this means this should go on the sequential folder.
+  // Refs: https://github.com/nodejs/node/issues/6758
+  // We need a timer like on the original issue thread.
+  // Otherwise the code will never reach our test case.
   _write(chunk, encoding, cb) {
-    setTimeout(cb, common.platformTimeout(10));
+    setImmediate(cb);
   }
 }
 
 const testStream = new StreamWritable();
 testStream.cork();
 
 for (let i = 1; i <= 5; i++) {
-  testStream.write(i, function() {
+  testStream.write(i, () => {
```

Comment:
`common.mustCall()`? Otherwise I think the `setImmediate()` above could also be dropped and the test would still pass.

Reply:
Yes, adding that will make it more clear that each call to `write()` must have a corresponding callback to run. The previous version of the test probably should have had it as well, I suppose. Anyway, added. Thanks!